cmd/env/env.go

```diff
@@ -141,13 +146,21 @@ func (e *OcEnv) Start() {
 	shell := os.Getenv("SHELL")
 
 	fmt.Print("Switching to OpenShift environment " + e.Options.Alias + "\n")
-	cmd := exec.Command(shell)
+	// ignore the following line in linter, only way to fix this is via setting a
+	// constant string for the exec.Command
+	cmd := exec.Command(shell) //nolint:gosec
```

Comment:
for these few items w/o a specific test disabled, is that because it wouldn't allow you to disable the specific test?

thx for adding the specific tests to the nosecs, too!

Reply:
@jaybeeunix  I have updated the 3 nolints and replaced them with a //#nosec line.. this allows us pinning to one specific issue